### ISSUE TITLE
investigate webhooks errors

### ISSUE THREAD

**Issue by kellihsf** [waffleio/waffle.io#3529]
a couple of users have reported 422 errors in GitHub on Waffle webhooks

**Comment by Workshop2** [comment]
Hello, is there any update on this?

**Comment by kellihsf** [comment]
hi @Workshop2, 

We're working on shipping a fix now. The issue should be in the "Needs Review" column shortly.

**Comment by Workshop2** [comment]
Great work, looking forward to this being resolved. Many thanks.

**Comment by Alisuehobbs** [comment]
@Workshop2, we just shipped a fix for this. Let us know if you need anything else!

**Comment by freakinruben** [comment]
@Alisuehobbs The `422` errors have definitely been reduced but I still see these errors happing, specifically when I'm calling the github deployment API frequent. 

In my use-case I call the github deployment API after a deployment to an environment is completed. I look up all the merged branches of the new release compared to the previous release:

`git log --merges --pretty=oneline alpha/old_version..alpha/new_version`

Then I call the github deployment api for each of the merged branches. This results in for example 20 remote branches being marked as deployed in alpha. Each deployment requires 2 calls to github, one to create the deployment and one to mark it as success.
I've already but delays of 1 second between each github API request which seem to reduce the `422` errors I see in the waffle hook.

Long story short, I want to mark a bunch of merged PRs as deployed, and the hook sometimes still fails, which means I need to go manually over the waffle board to make sure all the deployed cards are in the correct lane..

**Comment by Workshop2** [comment]
@kellihsf @Alisuehobbs sorry to say but we are seeing this issue again. I have messaged Kelly the error logs on Intercom
